src/utils/groupByUniqName.js

```diff
@@ -0,0 +1,17 @@
+import { isEmpty, chain, has } from 'lodash'
+
+export const groupByUniqName = (value, field = '') => {
+  const makeUniq = items => {
+    return chain(items)
+      .filter(item => has(item, field + '.name') || has(item, 'name'))
```

Comment:
Instead of doing this `||`, I think it might be a better approach to send the entire field path instead of a field that is supposed to have a `.name`.
So instead of using this function like this:
```js
groupByUniqName(jobs.data, 'metadata')
groupByUniqName(features, 'producer')
```
We'll do this instead:
```js
groupByUniqName(jobs.data, 'metadata.name')
groupByUniqName(features, 'producer.name')
```